ui-events/src/pointer/buttons.rs

```diff
@@ -6,43 +6,77 @@
 #[repr(u32)]
 pub enum PointerButton {
     /// No mouse button.
-    None,
+    None = 0,
     /// Primary button, commonly the left mouse button, touch contact, pen contact.
-    Primary,
+    Primary = 1,
     /// Secondary button, commonly the right mouse button, pen barrel button.
-    Secondary,
+    Secondary = 1 << 1,
     /// Auxiliary button, commonly the middle mouse button.
-    Auxiliary,
+    Auxiliary = 1 << 2,
     /// X1 (back) Mouse.
-    X1,
+    X1 = 1 << 3,
     /// X2 (forward) Mouse.
-    X2,
+    X2 = 1 << 4,
     /// Pen erase button.
-    PenEraser,
-    /// Other mouse button. This isn't fleshed out yet.
-    Other,
-    // If adding a new variant here, also update the `button_bit` and
-    // `fmt::Debug for PointerButtons` below.
+    PenEraser = 1 << 5,
+    /// Button 7
+    B7 = 1 << 6,
```

Comment:
I see that this is following the same format as X1/X2.
I'd probably want those to be Back/Forward instead, but that's not my call to make. And the web does weasel out of it by saying "typically".

I bring this up because I the names `X7`+ don't spark joy, although a better name doesn't spring to mind.